app/classifier/tasks/generic-editor.cjsx

```diff
@@ -57,6 +57,14 @@ module.exports = React.createClass
         </div>}
       {' '}
 
+      {if 'hide previous marks' in @props.project.experimental_tools
```

Comment:
By "property of the workflow", you mean it should be like `@props.workflow.enableHidePrevMarks` instead of `@props.task.enableHidePrevMarks`? If so, the back-end won't let me call `update()` on that.